src/globals/operations/findVersionByID.ts

```diff
@@ -73,10 +71,9 @@ async function findVersionByID<T extends TypeWithVersion<T> = any>(args: Argumen
     return null;
   }
 
-  // Clone the result - it may have come back memoized
-  result = JSON.parse(JSON.stringify(result));
 
-  result = sanitizeInternalFields(result);
+  // Clone the result - it may have come back memoized
+  let result = JSON.parse(JSON.stringify(results[0]));
```

Comment:
interesting that we have to leave this here. i guess it makes sense, just caught me by surprise

Reply:
Oh yea, to be honest, I'm not 1000% sure about this. We already `JSON.parse(JSON.stringify` this in the adapter. But that happens before it's memoized, not afterwards, so the comment might still be valid